rcl/test/memory_tools/test_memory_tools.cpp

```diff
@@ -20,22 +20,22 @@
  */
 TEST(TestMemoryTools, test_allocation_checking_tools) {
   size_t unexpected_mallocs = 0;
-  auto on_unexpected_malloc = (
-    [&unexpected_mallocs]() {
+  auto on_unexpected_malloc = [&unexpected_mallocs]()
+    {
       unexpected_mallocs++;
-    });
+    };
```

Comment:
These look very strange, would this not work instead?

```c++
  auto on_unexpected_malloc = [&unexpected_mallocs]() {
    unexpected_mallocs++;
  };
```

Or:

```c++
  auto on_unexpected_malloc =
    [&unexpected_mallocs]() {
      unexpected_mallocs++;
    };
```

For me, this should look like an initializer list, so like this:

```c++
foo = {
  'a',
  'b',
}
```

And not:

```c++
foo =
  {
    'a',
    'b',
  };
```

Reply:
Please try if the suggested alternative passes `uncrustify`. I really don't mind how it looks like.

Reply:
Please try if the suggested alternative passes `uncrustify` (I think I recall it doesn't). I really don't mind how it looks like.

Reply:
The first options doesn't pass. The second one does. Updated in https://github.com/ros2/rcl/pull/215/commits/dfd8ba25fcd9d440505e03b40161e264a9fd41e3#diff-b4937ca0f22682aded6f4aeaa9d0392bR23

Reply:
Thanks, lgtm.